cffi-toolchain.asd

```diff
@@ -0,0 +1,45 @@
+;;;; -*- Mode: lisp; indent-tabs-mode: nil -*-
+;;;
+;;; cffi-grovel.asd --- ASDF system definition for cffi-grovel.
+;;;
+;;; Copyright (C) 2007, Luis Oliveira  <loliveira@common-lisp.net>
```

Comment:
This should say cffi-toolchain rather than cffi-grovel. And I didn't write it, so I guess your name should be there? (We should remove these useless headers at some point in favor of a single LICENSE file, perhaps, but that's for another time.)